src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufactureService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufactureService manufactureService =
+                (ManufactureService) injector.getInstance(ManufactureService.class);
+
+        Manufacturer bmw = new Manufacturer(1L, "BMW", "Germany");
+        System.out.println("create manufacturer");
+        System.out.println(manufactureService.create(bmw) + System.lineSeparator());
+        System.out.println("get manufacturer");
+        System.out.println(manufactureService.get(bmw.getId()) + System.lineSeparator());
+        System.out.println("getAll manufacturers");
+        manufactureService.getAll().stream()
+                .forEach(System.out::println);
+        System.out.println();
+        bmw.setName("MINI");
+        System.out.println("update manufacturer");
+        System.out.println(manufactureService.update(bmw) + System.lineSeparator());
+        System.out.println("delete manufacturer");
+        System.out.println(manufactureService.delete(bmw.getId()) + System.lineSeparator());
```

Comment:
you can also display all drivers on the screen to show that the deletion works (Optional)